### PR TITLE
fix/PLF-8383 : Adjust error message position

### PR DESCRIPTION
**Pull request by aelbenney**
In order to well show the error message, we need to move out its div from the `drop zone div`.

### CODE REVIEW

File: webapp/portlet/src/main/resources/locale/portlet/social/NewsActivityComposer_fr.properties [hunk 1]
```diff
@@ -7,9 +7,10 @@ activity.composer.news.illustration=Upload the image that will appear as illustr
 activity.composer.news.error.BrowserNotSupported=Your current browser does not seem to support file upload.
 activity.composer.news.error.TooManyFiles=You can only upload up to {0} files at once.
 activity.composer.news.error.FileTooLarge=File size exceeds the {0} MB limit.
+activity.composer.news.error.FileTypeNotAllowed=Erreur de traitement, type de fichier non autorisé.
```
Comment (thomasdelhomenie) [inline webapp/portlet/src/main/resources/locale/portlet/social/NewsActivityComposer_fr.properties]:
```suggestion
activity.composer.news.error.FileTypeNotAllowed=Type de fichier non autorisé.
```

File: webapp/portlet/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue [hunk 2]
```diff
@@ -15,11 +15,14 @@
...
-                      :placeholder="$t('activity.composer.news.placeholderSummaryInput')"
-                      class="newsFormInput" type="text"/>
+            <div :class="newsFormExtendedClass" class="newsInputsRow">
+              <input id="newsTitle" v-model="newsActivity.title" :maxlength="max"
+                     :placeholder="$t('activity.composer.news.placeholderTitleInput')" class="newsFormInput"
+                     type="text">
+              <textarea v-show="extendedForm" id="newsSummary" v-model="newsActivity.summary" :maxlength="max"
+                        :placeholder="$t('activity.composer.news.placeholderSummaryInput')"
+                        class="newsFormInput" type="text"/>
+            </div>
```
Comment (thomasdelhomenie) [inline webapp/portlet/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue]: Shouldn't we change the structure of the DOM to have label and related input in the same div ? For example like in this tutorial: https://webdesign.tutsplus.com/tutorials/building-responsive-forms-with-flexbox--cms-26767
It seems more logic to me. WDYT ?

File: webapp/portlet/src/main/resources/locale/portlet/social/NewsActivityComposer_fr.properties (hunk 1 above)
Comment (aelbenney) [reply webapp/portlet/src/main/resources/locale/portlet/social/NewsActivityComposer_fr.properties]: okay

File: webapp/portlet/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue (hunk 2 above)
Comment (aelbenney) [reply webapp/portlet/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue]: ahh I thought that the refactor is only for the inputs with the image uploader! So I will do it for all the form like in the tuto

File: webapp/portlet/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue [hunk 3]
```diff
@@ -3,50 +3,59 @@
...
-            </a>
+            <div class="pinArticleContent">
+              <span class="uiCheckbox">
+                <input id="pinArticle" v-model="pinArticle" type="checkbox" class="checkbox ">
+                <span class="pinArticleLabel">{{ $t("activity.composer.news.pinArticle") }}</span>
+              </span>
+            </div>
+            <div class="newsFormActions">
+              <button id="newsPost" :disabled="postDisabled" class="btn btn-primary"> {{ $t("activity.composer.news.post")
+              }}
```
Comment (thomasdelhomenie) [inline webapp/portlet/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue]: Should not be on a new line.

Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue]: Done!

File: webapp/portlet/src/main/resources/locale/portlet/social/NewsActivityComposer_fr.properties (hunk 1 above)
Comment (thomasdelhomenie) [reply webapp/portlet/src/main/resources/locale/portlet/social/NewsActivityComposer_fr.properties]: Comment not considered.

**Pull request merged**